### PR TITLE
Unlock rendering of projects' main annotation layer onto the canvas

### PR DESCRIPTION
**Pull request by nirvn**
Proof of life:
image

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for f1761c9fe9e5592f909ae17d53b9c06cee1d20c8:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2139-4c21a4eab3d495bdb66a0f0200d71efb72216ced-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2139-4c21a4eab3d495bdb66a0f0200d71efb72216ced-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2139-4c21a4eab3d495bdb66a0f0200d71efb72216ced-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2139-4c21a4eab3d495bdb66a0f0200d71efb72216ced-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 7f14b6664b382b9796c77e23ccb417596e7e70fb:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2139-25cd81476eec7615fb15af57eb2f83934c896cc5-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2139-25cd81476eec7615fb15af57eb2f83934c896cc5-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2139-25cd81476eec7615fb15af57eb2f83934c896cc5-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2139-25cd81476eec7615fb15af57eb2f83934c896cc5-x86.apk)

File: src/core/qgsquick/qgsquickmapcanvasmap.cpp [hunk 1]
```diff
@@ -113,6 +114,11 @@ void QgsQuickMapCanvasMap::refreshMap()
     expressionContext << QgsExpressionContextUtils::projectScope( project );
 
     mapSettings.setLabelingEngineSettings( project->labelingEngineSettings() );
+
+    // render main annotation layer above all other layers
+    QList<QgsMapLayer *> allLayers = mapSettings.layers();
+    allLayers.insert( 0, project->mainAnnotationLayer() );
```
Comment (m-kuhn) [inline src/core/qgsquick/qgsquickmapcanvasmap.cpp:120]: Does that exist in every project?

Comment (nirvn) [reply src/core/qgsquick/qgsquickmapcanvasmap.cpp:120]: Yep.

**Review by m-kuhn — APPROVED** [review]

**Pull request merged**